src/plugins/minimal_scene/MinimalScene.hh

```diff
@@ -203,7 +214,7 @@ namespace plugins
     public: QSize textureSize = QSize(1024, 1024);
 
     /// \brief Flag to indicate texture size has changed.
```

Comment:
Set `true`, otherwise (in Metal at least) the scene dimensions are not rescaled to match the containing window and instead Qt interpolates the render texture and the aspect ratio is out. Setting `textureDirty = true` forces the resize call.